config/jobs/image-pushing/k8s-staging-capi-openstack.yaml

```
@@ -0,0 +1,32 @@
+postsubmits:
+  # this is the github repo we'll build from; this block needs to be repeated for each repo.
+  kubernetes-sigs/cluster-api-provider-openstack:
+    - name: post-capi-openstack-push-images
+      cluster: test-infra-trusted
+      annotations:
+        # this is the name of some testgrid dashboard to report to.
+        testgrid-dashboards: sig-cluster-lifecycle-image-pushes
+      decorate: true
+      branches:
+        - ^master$
+      spec:
+        containers:
+          - image: gcr.io/k8s-testimages/image-builder:v20190906-d5d7ce3
+            command:
+              - /run.sh
+            args:
+              # this is the project GCB will run in, which is the same as the GCR images are pushed to.
+              - --project=k8s-staging-capi-openstack
+              - --scratch-bucket=gs://k8s-staging-capi-openstack-scratch
```

This name has changed to `k8s-staging-capi-openstack-gcb`.

```suggestion
              - --scratch-bucket=gs://k8s-staging-capi-openstack-gcb
```